src/test/java/com/commercetools/sync/it/products/SphereClientUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.commercetools.sync.it.products;
+
+import com.commercetools.sync.commons.utils.ClientConfigurationUtils;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.queries.PagedQueryResult;
+
+import javax.annotation.Nonnull;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public class SphereClientUtils {
```

Comment:
yes, I based it on @jakubchrobak work but he did not merge his work in master so please consider it as WIP for now.